pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerStrategy.java

```diff
@@ -49,17 +49,15 @@ Optional<String> selectBroker(Set<String> candidates, BundleData bundleToAssign,
     /**
      * Create a placement strategy using the configuration.
      *
-     * @param conf
-     *            ServiceConfiguration to use.
      * @return A placement strategy from the given configurations.
      */
-    static ModularLoadManagerStrategy create(final ServiceConfiguration conf) {
+    static ModularLoadManagerStrategy create() {
```

Comment:
IMO, better to keep this in case we create some new `ModularLoadManagerStrategy` and needs configuration.

Reply:
The `ServiceConfiguration  conf` is used in the method `selectBroker` by `ModularLoadManagerStrategy`
https://github.com/apache/pulsar/blob/a812f29bab93a5e841a9b3c3190fca343fab6e71/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerStrategy.java#L31-L47